tutorials/easydeploy-grafana/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: Deploy ArgoCD on Scaleway Kubernetes clusters using Easy Deploy
+  description: Discover how to use Grafana on Scaleway Kubernetes clusters with Easy Deploy. Unlock the power of data visualization with Grafana, and follow our step-by-step instructions for deployment and management of your application.
+content:
+  h1: Deploy ArgoCD on Scaleway Kubernetes clusters using Easy Deploy
```

Comment:
H1 title is wrong, this tutorial is about Grafana

Reply:
```suggestion
  h1: Deploy Grafana on Scaleway Kubernetes clusters using Easy Deploy
```